app/bundles/LeadBundle/Helper/FormEventHelper.php

```diff
@@ -22,13 +22,13 @@
 class FormEventHelper
 {
     /**
-     * @param               $lead
+     * @param Lead          $lead
      * @param MauticFactory $factory
      * @param               $action
      * @param               $config
      * @param               $form
      */
-    public static function changePoints ($lead, MauticFactory $factory, $action, $config, $form)
+    public static function changePoints (Lead $lead, MauticFactory $factory, $action, $config, $form)
```

Comment:
@dongilbert Well `Lead::addPointsChangeLog` was already being called here before, so I imagine it is.